Submodule deps/fatrop contains modified content
diff --git a/deps/fatrop/CMakeLists.txt b/deps/fatrop/CMakeLists.txt
index 1092aef..0ff3790 100644
--- a/deps/fatrop/CMakeLists.txt
+++ b/deps/fatrop/CMakeLists.txt
@@ -15,7 +15,7 @@ if(NOT CMAKE_BUILD_TYPE)
 endif()
 
 # Set compilation flags for Release and Debug
-set(CMAKE_CXX_FLAGS_RELEASE "-O3 -march=native -DNDEBUG")
+set(CMAKE_CXX_FLAGS_RELEASE "-O3 -DNDEBUG")
 set(CMAKE_CXX_FLAGS_DEBUG "-g -O0")
 
 # Option to build tests (default is ON)
diff --git a/deps/fatrop/src/CMakeLists.txt b/deps/fatrop/src/CMakeLists.txt
index e130ff4..7025a80 100644
--- a/deps/fatrop/src/CMakeLists.txt
+++ b/deps/fatrop/src/CMakeLists.txt
@@ -34,7 +34,7 @@ set(FATROP_SOURCES
 )
 
 # Create the fatrop shared library
-add_library(fatrop SHARED ${FATROP_SOURCES})
+add_library(fatrop STATIC ${FATROP_SOURCES})
 
 # Set properties for the fatrop target
 set_target_properties(fatrop PROPERTIES
